src/helpers/transport/query.rs

```diff
@@ -190,14 +190,31 @@ pub struct IpaQueryConfig {
 impl Default for IpaQueryConfig {
     fn default() -> Self {
         Self {
-            per_user_credit_cap: 1,
+            per_user_credit_cap: 3,
```

Comment:
This is only used for the query processor test. Nonetheless, 1 is a special case and skips many attribution steps. Just to be on the safe side, set it to something other than 1.